boards/raspberrypi/rpi_pico2/rpi_pico2_rp2350a_hazard3_defconfig

```diff
@@ -0,0 +1,14 @@
+# This configuration is orthogonal to whether the Cortex-M33 or Hazard3 cores
+# are in use, but Zephyr does not support providing a qualifier-agnostic
+# _defconfig file.
+CONFIG_BUILD_OUTPUT_HEX=y
+CONFIG_BUILD_OUTPUT_UF2=y
+CONFIG_CLOCK_CONTROL=y
+CONFIG_CONSOLE=y
+CONFIG_GPIO=y
+CONFIG_RESET=y
+CONFIG_SERIAL=y
+CONFIG_SYS_CLOCK_HW_CYCLES_PER_SEC=150000000
```

Comment:
reminder @soburi this (CONFIG_SYS_CLOCK_HW_CYCLES_PER_SEC) needs to be updated to come from dts and get set at soc level, will nack any PR not doing this from next week onwards